src/clusterfuzz/_internal/system/archive.py

```diff
@@ -349,14 +346,15 @@ class ArchiveError(Exception):
 
 
 # pylint: disable=redefined-builtin
-def open(archive_path: str, file_obj: BinaryIO = None) -> ArchiveReader:
+def open(archive_path: str,
+         file_obj: Optional[BinaryIO] = None) -> ArchiveReader:
   """Opens the archive and gets the appropriate archive reader based on the
-  `archive_path`.
+  `archive_path`. If file_obj is not none, the binary file-like object will be
+  used to read the archive instead of opening the file.
```

Comment:
```suggestion
  `archive_path`. If file_obj is not none, the binary file-like object will be
  used to read the archive instead of opening `archive_path`.
```

For some reason, I'm not able to comment on two lines at once.